### PR TITLE
Add floor and ceil functions for float

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: rebo/src/stdlib/mod.rs [hunk 1]
```diff
@@ -399,3 +411,17 @@ fn file_read_to_string(name: String) -> Result<String, FileError> {
...
+mod test {
+    use crate::ReturnValue;
+    use crate::tests::test;
+
+    #[test]
+    fn test_float_rounding() {
+        test(r#"
+            let val = 3.14559;
+            let ceil_test = float::floor(val, 2);
+            print(ceil_test);
```
Comment (oberien) [inline rebo/src/stdlib/mod.rs]: To make sure you got the correct value, use `assert(1337 == b)`, which will make the test fail. Could you please test the `round` and `ceil` functions as well, and add more test numbers (one for each rounding-case)?

**Review by oberien — CHANGES_REQUESTED** [review]

File: rebo/src/stdlib/mod.rs (hunk 1 above)
Comment (ghost) [reply rebo/src/stdlib/mod.rs]: Will do 👍🏼

File: test.re [hunk 2]
```diff
@@ -329,3 +329,8 @@ assert(s.slice(1, 3) == "βγ");
 assert(s.slice(0, -1) == "αβγδε");
 assert(s.slice(1, -1) == "βγδε");
 assert(s.slice(-2, -1) == "ε");
+
+let val = 3.14545454;
+print(f"float::floor 1dp: {float::floor(val, 1)}");
+print(f"float::ceil  1dp: {float::ceil(val, 1)}");
+print(f"float::round 1dp: {float::round(val, 1)}");
```
Comment (oberien) [inline test.re]: No need for it in this file.

File: rebo/src/stdlib/mod.rs [hunk 3]
```diff
@@ -399,3 +411,23 @@ fn file_read_to_string(name: String) -> Result<String, FileError> {
...
+        test(r#"
+            // floor without decimals
+            let val = 3.14545454;
+            assert(float::floor(val, 0) == 3.);
+
+            // ceil without decimals
+            assert(float::ceil(val, 0) == 4.);
+
+            // round without decimals
+            assert(float::round(val, 0) == 3.);
```
Comment (oberien) [inline rebo/src/stdlib/mod.rs]: Could you please add more test cases? For example rounding a number can round down, round up, or not round at all. Add a test for every of those cases and think about similar cases for the other functions. Also try to round to different decimal points as well to check if that code is working as well. Try to capture as many different cases as possible.

**Review by oberien — CHANGES_REQUESTED** [review]

**Review by oberien — APPROVED** [review]
Thanks for the PR!

**Pull request merged**